### DISCUSSION TITLE
如何显示编译器的正常输出？

### DISCUSSION THREAD

**Discussion by Tawezy7** [xmake-io/xmake#4724] [General]
编译的警告和错误能够显示，在嵌入式开发我想输出FLASH RAM使用情况，于是在链接参数中加上-Wl,--print-memory-usage，编译器的警告和错误能够显示，但编译器的正常输出没有显示，虽然用xmake -v能够输出，但是我不想要输出命令，如图：
使用xmake -rv:
![无标题](https://github.com/xmake-io/xmake/assets/51575921/cd462476-9795-43f9-adcf-2722d579de62)
使用xmake -r:

![QQ截图20240209114833](https://github.com/xmake-io/xmake/assets/51575921/f44132bb-0e94-48a7-86f6-2106796eacf1)

**Comment by waruqi** [comment]
暂时不支持
